components/templates/ListOfTests.tsx

```diff
@@ -123,6 +123,19 @@ useSocket('delta_resolution', testResolution => {
     }
 })
 
+useSocket('delta_test', testDelta => {
+  let filteredTest = tests.find(
+    test => test.mother_test_id === testDelta.mother_test_id);
```

Comment:
Actually, looks like I made a mistake here, this should just `test_id`, as a `mother_test_id` would change the status of a lot of non related test 😧